tests/unit/design_document_tests.py

```diff
@@ -161,6 +161,32 @@ def test_update_design_document_with_encoded_url(self):
         remote_ddoc.fetch()
         self.assertEqual(remote_ddoc, ddoc)
 
+    def test_correct_design_document_partitioning(self):
```

Comment:
Let's split this into two test functions: `test_design_document_with_partitioned_true` and `test_design_document_with_partitioned_false`

Reply:
The split is done